packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="cypress" />
```

Comment:
is this fixing the typing issues?

Reply:
woops, no it wasn't working, forgot to remove it.